src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -67,6 +68,16 @@ public boolean hasPerson(Person person) {
         return persons.contains(person);
     }
 
+    public Person getPerson(String name) throws PersonNotFoundException {
```

Comment:
Would it be possible to take in a ComparablePerson instead of a string as parameter?